lib/openshift-rest-client.js

```diff
@@ -58,9 +57,16 @@ function getNames (resourceType) {
 const spec = JSON.parse(zlib.gunzipSync(fs.readFileSync(path.join(__dirname, 'specs', 'openshift-openapi-spec.json.gz'))));
 
 /**
-  @param {object} [settings] -
-  @returns {Promise} - Returns the Rest Client
-*/
+ * Builds the rest client based on provided or default kubernetes configuration.
+ *
+ * @param {object} [settings] - settings object for the openshiftClient function
```

Comment:
Looking good.  I would just add one more thing here `settings.config`,  this is an object that allows for passing in a custom config.  If you can,  maybe put a link to the advanced section of the readme

Reply:
Sounds good :+1: